### PR TITLE
Translate missing brazilian (PT-br) i18n json.

### PR DESCRIPTION
**Pull request by jeancx**
I translated some lines that are missing from the Brazilian language.

If i forgot any, let me know.

- [x] Ready for review.

### CODE REVIEW

File: packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json [hunk 1]
```diff
@@ -58,10 +58,10 @@
     "assign-products-to-channel": "Atribuir produtos ao canal",
     "assign-to-channel": "Atribuir ao canal",
     "assign-to-named-channel": "Atribuir a { channelCode }",
-    "assign-variant-to-channel-success": "",
-    "assign-variants-to-channel": "",
-    "auto-update-option-variant-name": "",
-    "auto-update-product-variant-name": "",
+    "assign-variant-to-channel-success": "Variações atribuídas ao canal com sucesso",
+    "assign-variants-to-channel": "Atribuir variante ao canal",
+    "auto-update-option-variant-name": "Atualizar automaticamente o nome das opções de variação",
```
Comment (jeancx) [inline packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json]:
```suggestion
    "auto-update-option-variant-name": "Atualizar automaticamente o nome das opções da variante",
```

File: packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json [hunk 2]
```diff
@@ -58,10 +58,10 @@
...
+    "auto-update-product-variant-name": "Atualizar automaticamente o nome dor produtos variáveis",
```
Comment (jeancx) [inline packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json]:
```suggestion
    "auto-update-product-variant-name": "Atualizar automaticamente o nome dos produtos variantes",
```

File: packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json [hunk 3]
```diff
@@ -104,13 +104,13 @@
     "no-selection": "Nenhuma seleção",
     "notify-remove-product-from-channel-error": "Não foi possível remover o produto do canal",
     "notify-remove-product-from-channel-success": "Produto removido com sucesso do canal",
-    "notify-remove-variant-from-channel-error": "",
-    "notify-remove-variant-from-channel-success": "",
+    "notify-remove-variant-from-channel-error": "Erro ao remover variação do canal",
```
Comment (jeancx) [inline packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json:107]:
```suggestion
    "notify-remove-variant-from-channel-error": "Erro ao remover variante do canal",
```

File: packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json [hunk 4]
```diff
@@ -104,13 +104,13 @@
...
+    "notify-remove-variant-from-channel-success": "Variação removida com sucesso do canal",
```
Comment (jeancx) [inline packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json:108]:
```suggestion
    "notify-remove-variant-from-channel-success": "Variante removida com sucesso do canal",
```

File: packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json [hunk 5]
```diff
@@ -608,22 +612,22 @@
...
-    "modification-not-settled": "",
-    "modification-recalculate-shipping": "",
-    "modification-settled": "",
-    "modification-summary": "",
-    "modification-updating-billing-address": "",
-    "modification-updating-shipping-address": "",
-    "modifications": "",
-    "modify-order": "",
-    "modify-order-price-difference": "",
+    "manually-transition-to-state": "Manually transition to state...",
```
Comment (jeancx) [inline packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json]:
```suggestion
    "manually-transition-to-state": "Manualmente mudar para estado...",
```

File: packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json [hunk 6]
```diff
@@ -608,22 +612,22 @@
...
+    "manually-transition-to-state-message": "Manually transition the order to another state. Note that order states are governed by rules which may prevent certain transitions.",
```
Comment (jeancx) [inline packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json]:
```suggestion
    "manually-transition-to-state-message": "Faça a transição manual do pedido para outro estado. Observe que os estados do pedido são regidos por regras que podem impedir certas transições.",
```

File: packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json [hunk 7]
```diff
@@ -608,22 +612,22 @@
...
+    "modification-adding-items": "Adding {count} {count, plural, one {item} other {items}}",
```
Comment (jeancx) [inline packages/admin-ui/src/lib/static/i18n-messages/pt_BR.json]:
```suggestion
    "modification-adding-items": "Adicionando {count} {count, plural, one {item} other {items}}",
```

**Review by jeancx — COMMENTED** [review]
Some fixes

**Comment by michaelbromley** [comment]
Thank you for your help!

**Pull request merged**